### PR TITLE
Games page image grid 

### PR DESCRIPTION
**Pull request by tmansuroglu**
Since games are not complete yet i didn't implement routing and game images. Also all game data inside i18n should be moved to an array. Or a new array should be created with game data.

Since we have 4 games I changed grid layout to fill screen

top padding added to title

#34 

Expectation 
expectation

placeholder game data is being used 

Output
screencapture-localhost-3000-games-2020-11-15-12_46_31

screencapture-localhost-3000-games-2020-11-15-12_50_38

### CODE REVIEW

File: src/components/gamepage/GamesPageImageGrid/index.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+// line 7-8-9 are mock datas
+import zelda from "../../../images/mockZeldaPoster.jpg";
```
Comment (ibrahimaljarad) [inline src/components/gamepage/GamesPageImageGrid/index.jsx]: i think you need to use something refers to image for naming, for example zeldaImg etc, for alll the images

**Review by ibrahimaljarad — APPROVED** [review]
i added one review only otherwise everything looks good

File: src/components/gamepage/GamesPageImageGrid/index.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+// line 7-8-9 are mock datas
```
Comment (louisrli) [inline src/components/gamepage/GamesPageImageGrid/index.jsx]: dont put comments in imports

File: src/components/gamepage/GamesPageImageGrid/index.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import zelda from "../../../images/mockZeldaPoster.jpg";
+import gta from "../../../images/mockGTAPoster.jpg";
+import destiny from "../../../images/mockDestinyPoster.jpg";
+
+const mockDataArr = [
```
Comment (louisrli) [inline src/components/gamepage/GamesPageImageGrid/index.jsx]: MOCK_DATA_ARR (this type of casing), and give it a better name. data for what?

Grand Theft Auto

also please make sure this is fixed by demo day

File: src/components/gamepage/GamesPageImageGrid/index.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  { name: "Grand theft auto", img: { src: gta, alt: "Grand theft auto" } },
+  { name: "Zelda", img: { src: zelda, alt: "Zelda" } },
+  { name: "Destiny", img: { src: destiny, alt: "Destiny" } },
+];
+
+const GamesPageImageGrid = () => {
+  const [t] = useTranslation();
+  const gameGridTitle = t("gamesPage.title");
```
Comment (louisrli) [inline src/components/gamepage/GamesPageImageGrid/index.jsx]: i would inline this if its not being used more than once

**Review by louisrli — COMMENTED** [review]
generally fine

File: src/locales/ar.json [hunk 5]
```diff
@@ -74,6 +74,9 @@
       ]
     }
   },
+  "gamesPage": {
```
Comment (ssinikoglu) [inline src/locales/ar.json]: I have a suggestion not only related with this code but games in general, would it be more neat if we put games inside of games Page?

**Review by ssinikoglu — APPROVED** [review]
Your code is clean and easy to understand.

File: src/locales/ar.json (hunk 5 above)
Comment (tmansuroglu) [reply src/locales/ar.json]: I was thinking of putting all game data in an array. But I will do it after all games are submitted. I need it for games page

File: src/components/gamepage/GamesPageImageGrid/index.jsx (hunk 1 above)
Comment (tmansuroglu) [reply src/components/gamepage/GamesPageImageGrid/index.jsx]: PTAL

File: src/components/gamepage/GamesPageImageGrid/index.jsx (hunk 2 above)
Comment (tmansuroglu) [reply src/components/gamepage/GamesPageImageGrid/index.jsx]: PTAL

File: src/components/gamepage/GamesPageImageGrid/index.jsx (hunk 3 above)
Comment (tmansuroglu) [reply src/components/gamepage/GamesPageImageGrid/index.jsx]: Name fixed. I will fix everything once all games are finished.

Comment (tmansuroglu) [reply src/components/gamepage/GamesPageImageGrid/index.jsx]: PTAL

File: src/components/gamepage/GamesPageImageGrid/index.jsx (hunk 4 above)
Comment (tmansuroglu) [reply src/components/gamepage/GamesPageImageGrid/index.jsx]: PTAL

**Comment by tmansuroglu** [comment]
There is weird test error. I will talk with Ammar about it.

File: src/components/gamespage/GamesPageImageGrid/index.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+import zeldaImg from "../../../images/mockZeldaPoster.jpg";
+import gtaImg from "../../../images/mockGTAPoster.jpg";
+import destinyImg from "../../../images/mockDestinyPoster.jpg";
+
+const MOCK_GAME_POSTER_DATA_ARR = [
+  { name: "Grand theft auto", img: { src: gtaImg, alt: "Grand theft auto" } },
```
Comment (louisrli) [inline src/components/gamespage/GamesPageImageGrid/index.jsx]: Grand Theft Auto

**Review by louisrli — APPROVED** [review]

File: src/components/gamespage/GamesPageImageGrid/index.jsx (hunk 6 above)
Comment (tmansuroglu) [reply src/components/gamespage/GamesPageImageGrid/index.jsx]: PTAL

**Comment by louisrli** [comment]
pass test for review

**Comment by louisrli** [comment]
or merge conflicts not sure

**Comment by tmansuroglu** [comment]
I will handle it

**Comment by tmansuroglu** [comment]
@Ammar-64 ready for merge

**Pull request merged**